libbeat/docs/shared-faq.asciidoc

```diff
@@ -0,0 +1,43 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/shared-faq.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[connection-problem]]
+=== Why doesn't my connection to Logstash work?
+
+You may have configured Logstash incorrectly. Make sure that:
+
+* The config file for you Beat specifies the correct port where Logstash is running. See
+{libbeat}/logstash-installation.html#logstash-setup[Setting Up Logstash]. 
+* The Elasticsearch output is commented out in the configuration file and the Logstash output is uncommented. 
+* The most recent Beats input plugin for Logstash is installed. See
+{libbeat}/logstash-installation.html#logstash-input-update[Updating the Beats Input Plugin for Logstash].
```

Comment:
Is beats-plugin configured in logstash? Beats will not connect to logstash lumberjack input plugin (Pro-Tip: beats input plugin is backwards compatible to lumberjack).